products/ceqr_survey/seeds/ceqr_variables.csv

```diff
@@ -1,13 +1,13 @@
-variable_type,variable_label,ceqr_category,type_ii_criteria_description
-zoning_districts,Zoning Districts,Zoning,"R5-R10, R1-R4, or (C or M)"
-coastal_risk_districts,Special Coastal Risk Districts,Zoning,"Any of 3 special purpose district fields in PLUTO contain a CR% value"
-e_designations_air,(E) Designations - air quality,(E) Designations,"Air Code = 1"
-e_designations_noise,(E) Designations - noise,(E) Designations,"Noise Code = 1"
-e_designations_hazmat,(E) Designations - hazmat,(E) Designations,"HazMat Code = 1"
-cats_permits,CATS Permits,Air Quality,"400 foot buffer of active and expired cats permits"
-state_facility_permits,State Facility Permits,Air Quality,"1,000 foot buffer"
-title_v_permits,Clean Air Act Title V Permits,Air Quality,"1,000 foot buffer"
-vent_towers,Vent Towers,Air Quality,"75 foot buffer"
-elevated_railways,Elevated Railways,Noise,"75 foot buffer"
-arterial_highways,Arterial Highways,Noise,"1,500 foot buffer"
-airport_contours,Airport Contours,Noise,"Intersects"
\ No newline at end of file
+variable_type,label,ceqr_category,survey_question,type_ii_criteria_description
+zoning_districts,Zoning Districts,Zoning,2A,"R5-R10, R1-R4, or (C or M)"
+special_coastal_risk_districts,Special Coastal Risk Districts,Zoning,2B,"Any of 3 special purpose district fields in PLUTO contain a CR% value"
+cats_permits,CATS Permits,Air Quality,10,"400 foot buffer"
+state_facility_permits,State Facility Permits,Air Quality,11,"1,000 foot buffer"
+title_v_permits,Clean Air Act Title V Permits,Air Quality,12,"1,000 foot buffer"
+vent_towers,Vent Towers,Air Quality,13,"75 foot buffer"
+e_designations_air,(E) Designations - Air Quality,Air Quality,14,"Air Code = 1"
+e_designations_noise,(E) Designations - Noise,Noise,15,"Noise Code = 1"
+arterial_highways,Arterial Highways,Noise,16,"75 foot buffer"
+elevated_railways,Elevated Railways,Noise,17,"1,500 foot buffer"
+airport_contours,Airport Contours,Noise,18,"Intersects 65db contour"
+e_designations_hazmat,(E) Designations - HazMat,HazMat,HZ1,"HazMat Code = 1"
```

Comment:
from the excel file it seemed like (E) Designations was an entire category. but are the values in the column `Questions` for that row in the sheet what informed the change here to put the 3 flags in Air Quality, Noise, and HazMat?

Reply:
E des is a single source which is why it is a single row in the excel. It has fields that seem to relate to multiple categories. In terms of ordering of question, `edes - noise` is with all the noise questions, `edes - air quality` is with all the air quality questions, so I assume they're meant to be grouped that way.

Reply:
Confirmed with @jackrosacker